apps/block_scout_web/test/block_scout_web/controllers/admin/dashboard_controller_test.exs

```diff
@@ -18,6 +18,7 @@ defmodule BlockScoutWeb.Admin.DashboardControllerTest do
       {:ok, conn: conn}
     end
 
+    @tag :skip
```

Comment:
It would be more clear to add  a tag specifically for account tests `@tag :account`

And then in test_helper, exclude tests w that tag

Reply:
agree, added and modified `mix.exs` file to include the `--exclude account` tag when the `mix test` command is used